rsconf/component/rsconf.py

```diff
@@ -29,6 +29,10 @@ def internal_build_compile(self):
             auth_f=nginx.CONF_D.join(PASSWD_SECRET_F),
             srv_d=jc.rsconf_db.srv_d,
             host_subdir=jc.rsconf_db.srv_host_d.basename,
+            kickstart_hosts=[
```

Comment:
I don't think this is a good idea. It should just be configuration. This assumes a lot about passwd_secret_f, which it shouldn't Nothing else assumes that except for setup_dev. It also requires the host to be added to rsconf before it has a base operating system. Rather, use a CIDR for the subnet with a list of special cases.

Reply:
Use trusted nets or just add config? https://github.com/radiasoft/rsconf/pull/535/commits/c2c36928e70baab657deeb5f723116cd7bcb88a2

Reply:
Add config. Let the user decide. This is security so using a default would be too much.

Reply:
40b6ce1